server-docs/src/test/java/org/neo4j/doc/server/rest/security/AuthenticationDocIT.java

```diff
@@ -86,18 +86,13 @@ public void successful_authentication() throws JsonParseException, IOException
         // Given
         startServerWithConfiguredUser();
 
-        // Document
-        RESTDocsGenerator.ResponseEntity response = gen.get()
-                .noGraph()
-                .expectedStatus( 200 )
-                .withHeader( HttpHeaders.AUTHORIZATION, challengeResponse( "neo4j", "secret" ) )
-                .get( userURL( "neo4j" ) );
+        HTTP.Response response = HTTP.withBasicAuth( "neo4j", "secret" ).POST( txCommitURL( "system" ), query( "SHOW USERS" ) );
```

Comment:
Is it not possible to use the old way of testing this?
Just changing the expected result